src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -55,38 +54,51 @@ export default class DiscussionArea {
   }
 
   /**
-   * Returns an array of the `DiscussionComment`s with timestamps near
-   * `timestampMs`. This returns an empty array if no elements are nearby.
-   *
-   * <p>A comment is nearby if it is within `TIME_TOLERANCE_MS`.
+   * Returns an array of the `DiscussionComment`s with the nearest time to
+   * `timeMs`. This typically returns an array with a single element, but
+   * if there are multiple comments the same distance away, they will all be
```

Comment:
Why do you need to keep track of all of the nearest comments? In `seek`, it only scrolls to the first comment.

Reply:
Correct.  For now, we only care about the single nearest.  

However, this PR also considers how we're gonna want to highlight comments in the future.  If there are 5 comments at the same timestamp, we want to highlight them all.

Scrolling uses a single one, highlighting will use multiple.

Reply:
Will add to PR description